docs/baseplate/core.rst

```diff
@@ -167,6 +133,57 @@ statsd :py:meth:`~!baseplate.core.SpanObserver.on_finish`.
 
 .. _convenience_methods:
 
+Edge Request Context
+--------------------
+
+The :py:class:`~baseplate.core.EdgeRequestContext` provides an interface into both
+authentication and context information about the original request from a user.  For edge
+services, it provides helpers to create the initial object and serialize the context
+information into the appropriate headers.  Once this object is created and attached to
+the context, Baseplate will automatically forward the headers to upstream services so they
```

Comment:
nit: we forward the information to downstream services (the request flows down to them)